docs/contents/components/forms/checkbox/accessibility.ja.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Checkbox
+description: "`Checkbox`は、ユーザーが複数の選択肢の中から複数の値を選択するための使用されるコンポーネントです。"
+package_name: "@yamada-ui/checkbox"
+with_description: true
+tab: アクセシビリティ
+---
+
+`Checkbox`は、アクセシビリティに関して[WAI-ARIA - Checkbox Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/checkbox)に従います。
+
+`FormControl`を使用しない場合は、`CheckboxGroup`に`aria-labeledby`を設定します。
+
+```tsx live=false
+<VStack gap="sm">
+  <Heading as="h3" size="lg">
+    キャラクター
+  </Heading>
+
+  <CheckboxGroup aria-labelledby="group">
+    <Checkbox value="孫悟空">孫悟空</Checkbox>
+    <Checkbox value="ベジータ">ベジータ</Checkbox>
+    <Checkbox value="フリーザ">フリーザ</Checkbox>
+  </CheckboxGroup>
+</VStack>
+```
+
+## キーボード操作
+
+| キー    | 説明                                               | 状態 |
+| ------- | -------------------------------------------------- | ---- |
+| `Tab`   | 無効ではないチェックボックスにフォーカスします。   | -    |
+| `Space` | フォーカスされたチェックボックスをチェックします。 | -    |
+
+## ARIAロールと属性
+
+| コンポーネント  | ロールと属性      | 使い方                                                                                                                  |
+| --------------- | ----------------- | ----------------------------------------------------------------------------------------------------------------------- |
+| `CheckboxGroup` | `role="group"`    | チェックボックスのグループであることを示します。                                                                        |
+|                 | `aria-labelledby` | `CheckboxGroup`が`FormControl`内にあり、`FormControl`に`label`または`Label`が設定されている場合、その`id`を設定します。 |
+| `Checkbox`      | `role="checkbox"` | チェックボックスであることを示します。                                                                                  |
```

Comment:
The `role="checkbox"` is not required. Because the `type="checkbox"` is already set.